process/interface.go

```diff
@@ -418,6 +418,7 @@ type BlockChainHookHandler interface {
 	TemporaryAccountsHandler
 	SetCurrentHeader(hdr data.HeaderHandler)
 	GetBuiltInFunctions() BuiltInFunctionContainer
+	NewAddress(creatorAddress []byte, creatorNonce uint64, vmType []byte) ([]byte, error)
```

Comment:
no need to redefine here - you could have a small interface at the genesis package.

Reply:
Ok, refactored a little bit the are, still using this same interface as to be able to reuse the same blockchain hook implementation used for creation of the processors. Will remain like this for the time being.